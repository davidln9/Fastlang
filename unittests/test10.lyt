
int main() {

	int x = 0;
	while (x < 5) {
		x = x + 1;
		println(x - 1 + get_message() + x);
	}
}


string get_message() {

	return "hello";
}
