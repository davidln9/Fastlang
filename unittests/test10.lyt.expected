0hello1
1hello2
2hello3
3hello4
4hello5
